src/routes/MapExplorer/MapExplorer.js

```diff
@@ -62,6 +62,8 @@ export class UnwrappedMapExplorer extends React.Component {
       groupIconLayer: null,
       hideDrawer: props.hideDrawer,
       hoveredObject: null,
+      canCreatePost: false,
```

Comment:
Is this maybe `creatingPost` vs `canCreatePost` ? I may be missing something but I think that naming signifies better how this state value is being used.